pipeline-model-extensions/src/main/java/org/jenkinsci/plugins/pipeline/modeldefinition/validator/DeclarativeValidatorContributor.java

```diff
@@ -65,122 +66,122 @@
      * Fallback for any unknown element type. Always returns true, cannot be overridden.
      */
     @CheckForNull
-    public final String validateElement(@Nonnull ModelASTElement element) {
+    public final String validateElement(@Nonnull ModelASTElement element, @CheckForNull FlowExecution execution) {
```

Comment:
Documenting the `execution` would be usefull, esp. when it's `null` and not and why that might matter.
Should be enough to do on a class level so you don't need to cut & paste to all the methods :)